src/sass/base/_section.scss

```diff
@@ -1,17 +1,22 @@
 .section {
-    padding-top: 66px;
-    padding-bottom: 66px;
+  padding-top: 66px;
+  padding-bottom: 66px;
   &__contacts {
     background-color: var(--background-color-2);
   }
-}
-
-@media screen and (min-width: 1280px) {
-  //  desktop
-}
-@media screen and (min-width: 768px) and (max-width: 1279px) {
-  // tablet
-}
-@media screen and (min-width: 450px) and (max-width: 767px) {
+  &__about {
```

Comment:
.section {
&__about {
    padding-top: 125px;
    padding-bottom: 39px;
    padding-left: 20px;
    padding-right: 20px;
  } }
  musisz miec to w section